admin_interface/locale/it/LC_MESSAGES/django.po

```diff
@@ -175,10 +179,36 @@ msgstr "codice"
 msgid "display"
 msgstr "visualizzazione"
 
+#: admin_interface/models.py
+msgid "Default: #333 (required fields, error messages)"
+msgstr ""
+
+#: admin_interface/models.py
+#, fuzzy
+#| msgid "background color"
+msgid "foreground color"
+msgstr "colore di sfondo"
```

Comment:
This is wrong, please leave it empty.

Reply:
it is a fuzzy entry, so won’t be displayed at run-time

(this happens when strings extraction finds a matching existing entry for a string that’s not the same, the previous or similar msgstr is added as a hint for translators but the entry is marked fuzzy)

Reply:
So nothing to delete in the end?